### PR TITLE
Chapter about DALEX for mlr3.

### PR DESCRIPTION
**Pull request by pbiecek**
An example prepared with the dev version of DALEX 1.2, which is currently only on GitHub. 
This chapter uses the FIFA data set. It is an ideal data set for instance-level explanations.
Everyone recognizes the individual instances from this dataset are.

### CODE REVIEW

**Comment by mllg** [comment]
This is great, thanks Przemysław! A few questions:

* What is the ETA for the cran release of 1.2? Should we wait for it to make it easier on our side?
* Do you want to be included in the list of authors of the book or at least mention that you wrote it somewhere in the DALEX section?
* Are you sure we want get copyright issues with the images? :worried:

**Comment by pbiecek** [comment]
Hi!
Thanks for quick feedback.

Ad 1. I plan to make a release at the end of the week (now I'm waiting for dependent packages to be accepted). I will send a message in this thread when DALEX 1.2 is on CRAN

Ad 2. That would be super nice. Though our contribution is small. For now, maybe we just add at the beginning of the chapter a short disclaimer "Who should be blamed: This chapter was prepared by ...". It was prepared with Szymon Maksymiuk, who created DALEXtra. He takes care of the integration of mlr3 with DALEX.

Ad 3. The FIFA data is relatively large (5000 cases, 43 variables) so some calculations can take up to a minute. If this is ok, then I will make these chunks live.

**Comment by pbiecek** [comment]
Quick update:

* DALEX v 1.2.1 is on CRAN
* All plots are now generated with runnable chunks of R code
* In this version there are only examples for a single model. We will add a second version with comparisons of multiple models

File: bookdown/08-interpretation.Rmd [hunk 1]
```diff
@@ -2,4 +2,247 @@
...
+|defending_standing_tackle  |37       |32            |26      |
+|defending_sliding_tackle   |26       |24            |29      |
+|goalkeeping_diving         |6        |7             |9       |
+|goalkeeping_handling       |11       |11            |9       |
+|goalkeeping_kicking        |15       |15            |15      |
+|goalkeeping_positioning    |14       |14            |15      |
+|goalkeeping_reflexes       |8        |11            |11      |
+
+
+In order to get a more stable model we remove four variables i.e. `nationality`, `overall`, `potential`, `wage_eur`.
```
Comment (mllg) [inline bookdown/08-interpretation.Rmd:88]: I found this confusing, these variables are not listed in the table.

File: bookdown/08-interpretation.Rmd [hunk 2]
```diff
@@ -2,4 +2,247 @@
...
+
+In the following sections, we will present subsequent methods available in the DALEX package based on a random forest model trained for football players worth prediction on the FIFA 20 data. We will show both methods analyzing the model at the level of a single prediction and the global level - for the whole data set.
+
+The structure of this chapter is the following:
+
+* In section \@ref(interpretability-data-fifa) we introduce the FIFA 20 dataset and then in section \@ref(interpretability-train-ranger) we train a random regression forest using the `r cran_pkg("ranger")` package.
+* Section \@ref(interpretability-architecture) introduces general logic beyond DALEX explainers.
+* Section \@ref(interpretability-dataset-level) introduces methods for dataset level model exploration.
+* Section \@ref(interpretability-instance-level) introduces methods for instance-level model exploration.
+* Section ... shows how to use the `DALEX` for cross-comparison of multiple models.
```
Comment (mllg) [inline bookdown/08-interpretation.Rmd:19]: Is this still in progress?

**Comment by mllg** [comment]
I've added some links and fixed some minor typos. I also added 2 comments in the review.

All in all, looks excellent. Can you add this disclaimer thing (or add yourselves to the list of authors)? After merging, I'll write a small blog post to announce the new chapter.

**Comment by pbiecek** [comment]
@mllg Thank you for the fixes. We have updated the table with players, list of sections and list of authors for this chapter.

**Comment by mllg** [comment]
Thanks!

**Pull request merged**